### DISCUSSION TITLE
about Sync Viewports

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#2999] [Q&A] [answered]
in https://www.itwinjs.org/sample-showcase/?group=Viewer+Features&sample=multi-viewport-sample&imodel=Metrostation+Sample

 open the Sync Viewports ,
 rotate the Top Viewport ,and the Bottom Viewport   can  Sync   rotate auto   success
 rotate  the Bottom Viewport ，the Top  Viewport   can  Sync   rotate auto ,but the  process is not smooth

**Comment by pmconne** [comment]
I fail to reproduce the problem. Am I overlooking something?
![2wayvpsync](https://user-images.githubusercontent.com/22944042/148536279-d53df342-6f68-4179-a5a9-632f4f14f3da.gif)

**Reply by pmconne** [reply]
I see - the problem occurs specifically when zooming in and out. We'll fix it.

**Comment by pmconne** [comment, marked as answer]
Fixed by #3002. Thanks for reporting the problem.
